### PR TITLE
add deploy to readme

### PR DESCRIPTION
**Pull request by JonasBa**

### CODE REVIEW

File: src/js/app.js [hunk 1]
```diff
@@ -19,8 +19,21 @@ const loadDefs = () => {
     })
 }
 
+function validateEmail(email) {
+  var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/
```
Comment (Haroenv) [inline src/js/app.js]: Is this change related to something else?

File: README.md [hunk 2]
```diff
@@ -2,6 +2,13 @@
 
 This is the homepage of https://community.algolia.com/.
 
+## Deploy
+
+```shell
+npm run build
+gulp deploy
```
Comment (Haroenv) [inline README.md]: are there environment variables needed?

Comment (JonasBa) [inline README.md]: No, not needed

File: src/js/app.js (hunk 1 above)
Comment (JonasBa) [inline src/js/app.js]: Just some quick email validation for newsletter signup

Comment (vvo) [inline src/js/app.js]: On a sidenote, was this required because we had many spam? I always feel like very complex regex at some point endsup blocking an email you don't want to be blocked (especially devs who are experts and can have emails that are edge cases).

Just curious here
